### PR TITLE
Fix TestPodGroupClean

### LINKED ISSUE #54: TestPodGroupClean is flaky
`TestPodGroupClean` in pkg/coscheduling/coscheduling_test.go seems to be flaky:

```
⇒  go test ./pkg/coscheduling/... -run TestPodGroupClean -count 50
--- FAIL: TestPodGroupClean (1.00s)
    --- FAIL: TestPodGroupClean/pod_belongs_to_a_podGroup (1.00s)
        coscheduling_test.go:567: fail to gc PodGroup in coscheduling: pod1
--- FAIL: TestPodGroupClean (1.00s)
    --- FAIL: TestPodGroupClean/pod_belongs_to_a_podGroup (1.00s)
        coscheduling_test.go:567: fail to gc PodGroup in coscheduling: pod1
--- FAIL: TestPodGroupClean (1.00s)
    --- FAIL: TestPodGroupClean/pod_belongs_to_a_podGroup (1.00s)
        coscheduling_test.go:567: fail to gc PodGroup in coscheduling: pod1
--- FAIL: TestPodGroupClean (1.00s)
    --- FAIL: TestPodGroupClean/pod_belongs_to_a_podGroup (1.00s)
        coscheduling_test.go:567: fail to gc PodGroup in coscheduling: pod1
FAIL
FAIL	sigs.k8s.io/scheduler-plugins/pkg/coscheduling	13.811s
FAIL
```

### PR DESCRIPTION
**Pull request by denkensk**
fix #54 
/cc @Huang-Wei

### CODE REVIEW

File: pkg/coscheduling/coscheduling_test.go [hunk 1]
```diff
@@ -536,10 +532,15 @@ func TestPodGroupClean(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			stop := make(chan struct{})
+			c := clock.NewFakeClock(time.Now())
```
Comment (Huang-Wei) [inline pkg/coscheduling/coscheduling_test.go:535]: Move this below `defer close(stop)`.

**Review by Huang-Wei — COMMENTED** [review]
Thanks @denkensk. Just a nit.

File: pkg/coscheduling/coscheduling_test.go (hunk 1 above)
Comment (denkensk) [reply pkg/coscheduling/coscheduling_test.go]: Done

**Pull request merged**